### PR TITLE
Update Main.java

### PR DESCRIPTION
**Pull request by NativeKB**
Pull request test

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -3,5 +3,6 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        System.out.println("Pull request test");
```
Comment (NativeKB) [inline src/main/java/org/example/Main.java:6]: added 1 line

Comment (Slawek84PL) [inline src/main/java/org/example/Main.java:6]: Test

Comment (Slawek84PL) [reply src/main/java/org/example/Main.java:6]: OK.

**Review by Slawek84PL — APPROVED** [review]
Great

**Pull request merged**